pygmo/core.cpp

```diff
@@ -885,8 +892,80 @@ BOOST_PYTHON_MODULE(core)
 
     moead_.def("get_seed", &moead::get_seed);
 
+    // Exposition of various structured utilities
```

Comment:
This and the other exposed classes/functions/methods need to be documented in the same way we document the rest of pygmo, with the sphinx napoleon syntax etc.

It would be good to have some tests on the python side as well (to test the exposition of the methods, not the hv functionality per-se).